x2gbfs/providers/cambio.py

```diff
@@ -0,0 +1,153 @@
+import json
+import logging
+from typing import Dict, Optional, Tuple
+
+from x2gbfs.gbfs.base_provider import BaseProvider
+from x2gbfs.util import get
+
+logger = logging.getLogger(__name__)
+
+
+class CambioProvider(BaseProvider):
+    """
+    The CambioProvider retrieves (static) stations and vehicleClasses from Cambio API.
+
+    """
+
+    DEFAULT_MAX_RANGE_ELECTRIC = 200000
+    DEFAULT_MAX_RANGE_COMBUSTION = 600000
+    STATIONS_URL = 'https://cwapi.cambio-carsharing.com/opendata/v1/mandator/{city_id}/stations'
+    VEHICLE_TYPES_URL = 'https://cwapi.cambio-carsharing.com/opendata/v1/mandator/{city_id}/vehicles'
+
+    def __init__(self, feed_config):
+        self.city_id = feed_config['provider-info']['city_id']
+        self.config = feed_config
+
+    def _all_stations(self) -> dict:
+        response = get(self.STATIONS_URL.replace('{city_id}', self.city_id))
+        response.raise_for_status()
+        return response.json()
+
+    def _all_vehicle_types(self) -> dict:
+        response = get(self.VEHICLE_TYPES_URL.replace('{city_id}', self.city_id))
+        response.raise_for_status()
+        return response.json()
+
+    def load_stations(self, default_last_reported: int) -> Tuple[Dict, Dict]:
```

Comment:
If it runs at Python 3.9+, you can move to `-> tuple[dict, dict]`.

Reply:
Implemented with 8a7ef9531597819612f76c1eccc10b369f758ecd